src/core/framegraphes/forwardrenderer.cpp

```diff
@@ -189,6 +195,18 @@ using namespace Kuesa;
     Holds the size of the external render target.
 */
 
+/*!
```

Comment:
Was this indented with Ctrl+E,R in QtCreator ?

Reply:
I just copied the formatting of the other property docs